typings/webgme.d.ts

```diff
@@ -2488,10 +2595,10 @@ declare namespace GmeClasses {
          */
         makeCommit: {
             (branchName: string, parents: GmeStorage.CommitHash[],
-                rootHash: Core.ObjectHash, coreObjects: Core.DataObject,
+                rootHash: Core.ObjectHash, coreObjects: object,
```

Comment:
I prefer a better type than `object`

Reply:
In the short term, you can even just use `type` to alias it for now until we can actual define a meaningful type.